app/modules/sightings/parameters.py

```diff
@@ -85,3 +88,21 @@ def add(cls, obj, field, value, state):
                 ret_val = True
 
         return ret_val
+
```

Comment:
have we "decided"(?) that `op=add` and `op=replace` should both be [supported/identical](https://github.com/WildMeOrg/houston/blob/main/app/modules/annotations/parameters.py#L30-L33) in the case of non-list values?   if so, i guess we need to throw `add()` in here yes?

Reply:
We have and what should happen is that replace just calls add and that implements it, see asset_groups patch

Reply:
In asset group add actually calls replace, I'll mimic that unless there is a reason to switch both to the other way around.